ouroboros-network/src/Ouroboros/Network/Chain.hs

```diff
@@ -186,11 +183,12 @@ addBlock b c = assert (validExtension c b) $
                c :> b
 
 pointOnChain :: HasHeader block => Point block -> Chain block -> Bool
-pointOnChain p Genesis        = p == genesisPoint
-pointOnChain p (c :> b)
-  | pointSlot p >  blockSlot b = False
-  | pointSlot p == blockSlot b = pointHash p == BlockHash (blockHash b)
-  | otherwise                  = pointOnChain p c
+pointOnChain GenesisPoint               _       = True
+pointOnChain (BlockPoint _ _)           Genesis = False
+pointOnChain p@(BlockPoint pslot phash) (c :> b)
+  | pslot >  blockSlot b = False
+  | pslot == blockSlot b = phash == blockHash b
+  | otherwise             = pointOnChain p c
```

Comment:
Can you align this last case on `=` to appease my OCD :slightly_smiling_face:

Reply:
Oops. Will do